sa/type-converter.go

```diff
@@ -38,6 +38,16 @@ func (tc BoulderTypeConverter) ToDb(val interface{}) (interface{}, error) {
 		return string(t), nil
 	case core.OCSPStatus:
 		return string(t), nil
+	case *core.AcmeURL:
```

Comment:
(It's definitely wrong because, as you see, gorp has turned it into a varchar, not a blob.)